CodeEdit/Features/Git/SourceControlManager.swift

```diff
@@ -43,14 +48,26 @@ final class SourceControlManager: ObservableObject {
     }
 
     /// Refresh all changed files and refresh status in file manager
-    func refresAllChangesFiles() async {
+    func refreshAllChangedFiles() async {
         do {
-            var changedFiles: [CEWorkspaceFile] = []
+            var fileDictionary = [URL: CEWorkspaceFile]()
 
+            // Process changed files
             for item in try await gitClient.getChangedFiles() {
-                changedFiles.append(.init(url: item.fileLink, changeType: item.changeType))
+                fileDictionary[item.fileLink] = CEWorkspaceFile(
+                    url: item.fileLink,
```

Comment:
Not entirely sure if this applies, but wouldn't URI be a better name to use when dealing with local files?

Reply:
Instead of `fileLink` or `url`? It is `url` in `CEWorkspaceFile`.

Reply:
Instead of URL. But this if fileLink refers to a typical URL link or just a local file. Apparently though Swift uses the name URL to also refer to local files, so we can probably just keep this as URL.